requirements.txt

```diff
@@ -1,39 +1,87 @@
 alembic==0.9.8
-bugsnag==3.4.2
+amqp==1.4.9
+aniso8601==3.0.2
+anyjson==0.3.3
+astroid==2.2.5
+attrs==19.1.0
+autopep8==1.4.4
+billiard==3.3.0.23
 blinker==1.4
+bugsnag==3.4.2
 celery==3.1.17
+certifi==2019.3.9
+chardet==3.0.4
+Click==7.0
 coverage==4.5.1
+coveralls==1.8.0
+decorator==4.4.0
+docopt==0.6.2
+eventlet==0.25.0
+flake8==3.5.0
 Flask==0.12.2
 Flask-Cors==3.0.4
-Flask-JSON==0.3.2
-Flask-Script==2.0.6
 Flask-GraphQL==1.4.1
+Flask-JSON==0.3.2
 Flask-Mail==0.9.1
+Flask-Script==2.0.6
+flask-socketio==4.1.0
+Flask-Testing==0.7.1
 google-api-python-client==1.6.7
-graphene-sqlalchemy==2.0.0
 graphene==2.1
+graphene-sqlalchemy==2.0.0
+graphql-core==2.2
```

Comment:
There seem to be additional dependencies added on here...Kindly ensure to install only the dependencies you need for this task.